sensor_dht.py

```diff
@@ -0,0 +1,22 @@
+from sensor import Sensor
+import time
+
+
+class SensorDht(Sensor):
+    def __init__(self):
+        super().__init__()
+
+    def read_value(self, sensor_type):
+        n = 1
+        while n < 6:
+            from_first_sensor = self.dht_devices[0].locals()[sensor_type]  # podmiana nazwa metody z stringa na metode
+            from_second_sensor = self.dht_devices[1].locals()[sensor_type]
+            if from_first_sensor is not None and from_second_sensor is not None:
+                value = (from_first_sensor + from_second_sensor) / 2
+                self.sensors_indications[value] = value
```

Comment:
to działa? bo klasa SensorDht nie ma pola sensors_indications

Reply:
jest brane z Sensor bo jest super wiec bierze z jego inita to czego tu nie znajdzie